examples/concertbot/actions/actions.py

```diff
@@ -49,3 +49,19 @@ def run(self, dispatcher, tracker, domain):
         venues = tracker.get_slot("venues")
         dispatcher.utter_message(text=f"venues from slots: {venues}")
         return []
+
+
+class ActionSetMusicPreference(Action):
+    def name(self):
+        return "action_set_music_preference"
+
+    def run(self, dispatcher, tracker, domain):
+        """Sets the slot 'likes_music' to true/false dependent on whether the user
+        likes music"""
+        intent = tracker.latest_message["intent"].get("name")
+
+        if intent == "affirm":
+            return [SlotSet("likes_music", True)]
+        elif intent == "deny":
+            return [SlotSet("likes_music", False)]
+        return []
```

Comment:
Doesn't look like this is referenced anywhere - and it should be handled by the `from_intent` mapping?

Reply:
Oh now I see, this isn't for forms this is for contextual conversations.